contrib/mesos/pkg/election/etcd_master.go

```diff
@@ -92,9 +92,11 @@ func (e *etcdMasterElector) extendMaster(path, id string, ttl uint64, res *etcd.
 	// We don't handle the TTL delete w/o a write case here, it's handled in the next loop
 	// iteration.
 	opts := etcd.SetOptions{
-		TTL:       time.Duration(ttl)*time.Second,
+		TTL:       time.Duration(ttl) * time.Second,
 		PrevValue: "",
 		PrevIndex: res.Node.ModifiedIndex,
+		// it was compareAndSwap
+		PrevExist: etcd.PrevExist,
```

Comment:
I removed this as part of testing in the last commit trying to figure out if it was something I was doing, same as the one afterwards.